{"name": "Тест авторизации курьера с несуществующим логином", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидали сообщение \"Учетная запись не найдена\", а получили <bound method Response.json of <Response [404]>>\nassert {'code': 404,...ь не найдена'} == {'message': '...ь не найдена'}\n  \n  Omitting 1 identical items, use -vv to show\n  Left contains 1 more item:\n  {'code': 404}\n  Use -v to get more diff", "trace": "self = <tests.test_login_courier.TestLoginCourier object at 0x00000244E9E89E00>\ncreate_courier_and_delete = {'first_name': 'cdugmextib', 'id': 517944, 'login': 'pmarplzxnb', 'password': 'mcmlyaiwzx'}\n\n    @allure.title('Тест авторизации курьера с несуществующим логином')\n    # тест, который проверит:\n    # если авторизоваться под несуществующим пользователем, запрос возвращает ошибку;\n    def test_create_and_login_courier_with_invalid_login(self, create_courier_and_delete):\n        _, password, _, _ = create_courier_and_delete\n        payload = {\"login\": 'invalid_login', \"password\": password}\n        response, status = CourierMethods().login_courier(payload)\n        assert status == 404, f'Ожидали 404, получили {status}'\n>       assert response.json() == {\"message\": \"Учетная запись не найдена\"}, f'Ожидали сообщение \"Учетная запись не найдена\", а получили {response.json}'\nE       AssertionError: Ожидали сообщение \"Учетная запись не найдена\", а получили <bound method Response.json of <Response [404]>>\nE       assert {'code': 404,...ь не найдена'} == {'message': '...ь не найдена'}\nE         \nE         Omitting 1 identical items, use -vv to show\nE         Left contains 1 more item:\nE         {'code': 404}\nE         Use -v to get more diff\n\ntests\\test_login_courier.py:49: AssertionError"}, "start": 1747162869726, "stop": 1747162869968, "uuid": "d8155ecf-2f76-4c2d-bae9-5502b02d4c3d", "historyId": "02e9a3c7e9409f8b7c21e8ad02a88044", "testCaseId": "02e9a3c7e9409f8b7c21e8ad02a88044", "fullName": "tests.test_login_courier.TestLoginCourier#test_create_and_login_courier_with_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "WIN-E0CER839EN3"}, {"name": "thread", "value": "4664-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}